aspnetcore/client-side/blazor/index.md

```diff
@@ -21,15 +21,15 @@ To learn more about Blazor's technological backbone, see the following documenta
 
 ## Get started
 
-For an overview of Blazor features, see [Blazor at the .NET website](https://www.microsoft.com/net/web/blazor). For development setup instructions and a quick blast of Blazor, see [Get started with Blazor in 10 minutes](https://www.microsoft.com/net/web/blazor/get-started). You can find answers to common questions in the [Blazor FAQ](https://github.com/aspnet/Blazor/wiki/FAQ). To chat live with other rockstars about Blazor, visit [Gitter](https://gitter.im/aspnet/Blazor) or [Slack](https://aspnetcore.slack.com/messages/C958WJP2N/) ([signup](http://tattoocoder.com/aspnet-slack-sign-up/)).
+[Get started with Blazor in 10 minutes](https://www.microsoft.com/net/web/blazor/get-started)
 
 ## Explore preview Blazor documentation
 
 \[Build your first Blazor app](xref:client-side/blazor/first-app)  
 DESCRIPTION
 
-\[Introduction to Blazor](xref:client-side/blazor/introduction/index)  
-DESCRIPTION
+[Introduction to Blazor](xref:client-side/blazor/introduction/index)  
+Explains how Blazor runs in the browser to execute C#/Razor code with WebAssembly and the Mono runtime.
```

Comment:
I recommend providing a link to the Mono site: http://www.mono-project.com/. Devs coming to Blazor without a .NET background will wonder what Mono is.

Reply:
We can drop "Mono" here per `@danroth27`'s earlier remark and just say ".NET runtime." Thanks for calling it out.